### PR TITLE
A small Sphinx comments

### PR DESCRIPTION
**Pull request by vahev**
This PR adds just a little of Sphinx-ready comments for utils module

### CODE REVIEW

**Review by elohimf — CHANGES_REQUESTED** [review]
The comments are missing the argument explanations and types. For example, clean_filenames takes a single arg, filenames, which is a list of strings. Consider using :param, :type and :returns for this.

**Comment by vahev** [comment]
@elohimf check it out, I've already done the changes that you mentioned before.
Also, I added another commit in which the Pyroute's installation on docker is added.

File: pyroute/utils.py [hunk 1]
```diff
@@ -85,30 +133,41 @@ def get_shared_attrs(objs):
...
     def clear_dunders(methods):
         """
-        This method takes a list of methods and filters out those 
-        that are Python's 'dunder' methods
+        This method takes a list of methods and filters out those
+        that are Python's 'dunder' methods.
+
+        :param methods: A list of Methods and Filters
+        :type methods: List <String>
+        :returns List: A list of dunder methods
```
Comment (elohimf) [inline pyroute/utils.py]: I think you mean "A list of non-dunder methods" here, as this method deletes all dunder methods.

**Review by elohimf — CHANGES_REQUESTED** [review]
Also, you should use interactive rebase to squash your commits into one, to keep everything tidy.

**Comment by vahev** [comment]
@elohimf I've already put my last commits into a single one. Also made the small change to utils file.

**Review by elohimf — APPROVED** [review]

**Pull request merged**